assets/dev/js/frontend/handlers/accessibility/nested-title-keyboard-handler.js

```diff
@@ -9,11 +9,15 @@ export default class NestedTitleKeyboardHandler extends Base {
 		this.focusableElementSelector = 'audio, button, canvas, details, iframe, input, select, summary, textarea, video, [accesskey], [contenteditable], [href], [tabindex]:not([tabindex="-1"])';
 	}
 
+	getWidgetNumber() {
+		return this.$element.find( '> .elementor-widget-container > .e-n-tabs, > .e-n-tabs' ).attr( 'data-widget-number' );
+	}
+
 	getDefaultSettings() {
 		return {
 			selectors: {
-				itemTitle: '.e-n-tab-title',
-				itemContainer: '.e-n-tabs-content > .e-con',
+				itemTitle: `[id*="e-n-tab-title-${ this.getWidgetNumber() }"]`,
+				itemContainer: `[id*="e-n-tab-content-${ this.getWidgetNumber() }"]`,
```

Comment:
Is the substring selector really required here?
I which cases will we want to get all the items that contain an ID with `e-n-tab-content-`?

Reply:
I prefer to handle it in this way, so that we can never have a clash with an inner nested tab.
I think this is the safest way to do it.

Wdyt?